src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java

```diff
@@ -785,4 +785,66 @@ public void testBannerMessages(){
         assertEquals("OK", status);
         
     }
+    
+    @Test
+    public void testLicenses(){
+        
+        String pathToJsonFile = "scripts/api/data/license.json";
+        Response addLicenseResponse = UtilIT.addLicense(pathToJsonFile);
+        addLicenseResponse.prettyPrint();
```

Comment:
Does this prettyPrint something to the STDOUT? I haven't yet figured out how to run integration tests so I couldn't check. Anyway, I would prefer not to write anything to the stdout, although I know this happens in a lot of Dataverse unit tests already. 

If it does not write anything to the stdout what does it do though?

Reply:
@janvanmansum You can run these integration tests using `mvn test -Dtest=AdminIT`. The prettyPrint does indeed print to STDOUT, which I also dislike. But in all the integration tests of the admin api the prettyPrint is used, so it's more consistent to adhere to the practice.